libcontainer/configs/mount.go

```diff
@@ -22,6 +28,9 @@ type Mount struct {
 	// Relabel source if set, "z" indicates shared, "Z" indicates unshared.
 	Relabel string `json:"relabel"`
 
+	// Extensions are additional flags that are specific to runc.
+	Extensions int `json:"extensions"`
```

Comment:
While I love the use of bits for this, I feel like it's more Go-like to use a slice of integers (which is parsed from a slice of strings) or something like that (so it makes sense when you look at the JSON).